cli/bpmetadata/schema/generate.go

```diff
@@ -41,6 +41,16 @@ func GenerateSchema() ([]byte, error) {
 	r := &jsonschema.Reflector{}
 	s := r.Reflect(&bpmetadata.BlueprintMetadata{})
 	s.Version = "http://json-schema.org/draft-07/schema#"
+
+	// defaultValue was defined as interface{} and has changed to
```

Comment:
Couldn't we keep it as Any?

https://protobuf.dev/programming-guides/proto3/#any

Reply:
I don't think we can because marshaling/unmarshaling to/from Any can only be done with/to proto messages and not primitive types AFAIK. Here we have the default value being returned as an interface{} and we need to convert it to some valid proto type and string seems to be the most appropriate type.

Open to other suggestions if you have any.

Reply:
I did some digging and it looks like the value type might be a good choice. We can also convert from interface{} to value easily using https://pkg.go.dev/google.golang.org/protobuf@v1.31.0/types/known/structpb#NewValue. WDYT?

Reply:
Thanks a bunch. That worked beautifully.